### ISSUE TITLE
[COGNITION]: Need a way to reach the main Outreach & Events page in the left navigation

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#900]
## Description

The left navigation doesn't include a way to reach the primary (first) Outreach & Events page. Screenshot attached below.

Tagging @peggygannon for guidance on what the expand/collapse section label and landing page link text should be.

## Point of Contact

**VFS Point of Contact:** _Trevor, Peggy_

## Acceptance Criteria

* As a user, I want to hear, see, and be able to interact with the Outreach & Events main page link

## Environment
* MacOS Mojave
* Chrome latest

## WCAG or Vendor Guidance (optional)
* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-09 at 3.49.49 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/95565157-f188-4e3c-ab96-0637ffbc5478)

**Comment by jefflbrauer** [comment]
Hi @1Copenut -  This follows the design pattern across the current va.gov. Unless we receive direction from @JeffBarnesUSDS we will not address this issue.

**Comment by jefflbrauer** [comment]
@1Copenut This follows the design pattern across the new va.gov per the design system. Unless @andrewlewandowski says otherwise, we won't address this specifically for the outreach product.

**Comment by 1Copenut** [comment]
@jefflbrauer I took a look at the [sidenav docs](https://design.va.gov/components/sidenav) in the VA design system. I'll defer to @andrewlewandowski about the right approach and happy to close this ticket if that's the right call.

**Comment by andrewlewandowski** [comment]
@1Copenut - Let's close this ticket for now. @rtwell is working on updating all of our global navigation, to fix a handful of usability problems and some possible 508 issues. He will work directly with you on designing our new navigation patterns so they are all 508 compliant and accessible!

**Comment by 1Copenut** [comment]
@andrewlewandowski 10-4. I'll move this to close and work with Ryan at the global scale. Thank you!
